lint/backend.py

```diff
@@ -60,12 +60,12 @@ def run_tasks(tasks, next):
 def get_lint_tasks(linters, view, hit_time):
     for (linter, regions) in get_lint_regions(linters, view):
 
-        def make_task(region):
+        def make_task(linter, region):
```

Comment:
This is actually unexpected for me as well, but I honestly would have defined the funtion outside of the loop anyway. Currently you create a new function for each lint and that's really not necessary. Especially when you can't bind the used variables in a loop.